keps/sig-network/3668-reserved-service-nodeport-range/README.md

```diff
@@ -305,7 +305,9 @@ This feature doesn't modify the cluster behavior, only the order on which dynami
 
 #### GA
 
-- No issues reported during two releases.
+- No issues reported during one release. This is because the feature is built on the top of 
+  [ServiceIPStaticSubrange](https://kep.k8s.io/3070), which has been GA since v1.26. 
+  Consequently, there is no need for any additional probation period.
```

Comment:
Can't comment on unchanged lines, so commenting here:

L367: have those metrics been added? If so, please update, if not, I would like to see them added before approving GA graduation.

Reply:
Yes, those metrics have been added in 1.27. 
https://github.com/kubernetes/kubernetes/pull/114418/files#diff-5532df35d48ad268e48b9761bd16d6c1204a7fa05cfd97afffcf47008ba012f7R31

Reply:
Can you please update the wording in the PRR to reflect that?

Reply:
It's updated.